packages/clay-core/src/tree-view/TreeViewItem.tsx

```diff
@@ -725,38 +724,19 @@ export function ItemStack({
 		<Layout.ContentRow {...otherProps}>
 			{expandable && !loading && (
 				<Layout.ContentCol>
-					<Button
-						aria-controls={`${item.key}`}
-						aria-expanded={expandedKeys.has(item.key)}
+					<span
+						aria-hidden="true"
 						className={classNames(
 							'component-expander',
 							expanderClassName,
 							{
 								collapsed: expandedKeys.has(item.key),
 							}
 						)}
-						disabled={
-							typeof expanderDisabled === 'undefined'
-								? disabled
-								: expanderDisabled
-						}
-						displayType={null}
-						monospaced
-						onClick={(event) => {
-							event.stopPropagation();
-
-							if (onLoadMore) {
-								onLoadMore();
-							} else {
-								toggle(item.key);
-							}
-						}}
 						tabIndex={-1}
```

Comment:
We can remove `tabindex` now since it's no longer a button. It's no longer included in the tab order.